content/blog/2020/05/2020-05-07-docker-windows-agents.adoc

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "Windows Docker Agent Images"
+tags:
+- announcement
+- docker
+- platform-sig
+author: slide_o_mix
+opengraph:
+  image: /images/docker/dockerJenkins_social.png
+---
+
+We would like to announce the availability of Windows agent images for Docker.
+
+image:/images/docker/dockerJenkins.png[Jenkins and Docker, role=center, float=right, height=224]
+
+== New images
+
+* link:https://hub.docker.com/repository/docker/jenkins/agent[jenkins/agent] is a basic agent which bundles the agent.jar for agent <= => master communication. This is most useful as a base image for other images.
+
+* link:https://hub.docker.com/repository/docker/jenkins/inbound-agent[jenkins/inbound-agent] is an agent that is based on the jenkins/agent image above. It provides a wrapper script written in PowerShell to help specify the parameters to agent.jar.
+
```

Comment:
Another plugins reference on this line

Reply:
I don't see a plugin reference in this part.